src/audio/kpb.c

```diff
@@ -893,6 +897,130 @@ static int kpb_reset(struct comp_dev *dev)
 	return ret;
 }
 
+#ifdef KPB_HIFI3
+static void kpb_micselect_copy(struct comp_dev *dev, struct comp_buffer __sparse_cache *sink,
+			       struct comp_buffer __sparse_cache *source, size_t size,
+			       uint32_t channels)
```

Comment:
since this function is do the audio stream copy, there is a function audio_stream_copy in audio_stream_copy.h, we can just call that function to replace this function.